runtime/src/bank.rs

```diff
@@ -2113,6 +2113,10 @@ impl Bank {
     pub fn clean_accounts(&self) {
         self.rc.accounts.accounts_db.clean_accounts();
     }
+
+    pub fn clean_dead_slots(&self) {
+        self.rc.accounts.accounts_db.handle_dead_slots();
```

Comment:
nits: `handle_` sounds like a post-process after some action for me and I don't think we should align with `handle_reclaims()`. How about `process_dead_slots`?, then we can avoid odd verb-swapping here (`clean_` and `handle_`) without name conflict. :)